Codeinwp/neve
# handle WP_Filesystem not available

### Summary
Should handle cases where WP_Filesystem is not available. 


### Will affect visual aspect of the product

NO

### Test instructions


- Inside `wp-config.php` add the following line `define( 'FS_METHOD', 'ftpext' );`;
- Make sure you remove `composer.lock` from the theme folder and run `composer install` as this also needs changes in the `themeisle-onboarding` repo;
- The WP_Filesystem method will not be available so the onboarding should display a notice;
- This should not happen if WP_Filesystem is available;


Closes Codeinwp/neve-pro-addon#693.
